src/utils/convert.js

```diff
@@ -162,19 +162,22 @@ const renderStep = (type, definition) => {
   ['inputFields', 'outputFields'].forEach(key => {
     const fields = definition.operation[key];
     if (Array.isArray(fields) && fields.length > 0) {
-      // Backend converter always put custom field function at the end of the array
-      const func = fields[fields.length - 1];
-      if (func && func.source) {
-        const args = func.args || ['z', 'bundle'];
-        const funcName = `get${_.upperFirst(key)}`;
-        functionBlock.push(
-          `const ${funcName} = (${args.join(', ')}) => {${func.source}};`
-        );
-
-        exportBlock.operation[key][fields.length - 1] = makePlaceholder(
-          funcName
-        );
-      }
+      // Godzilla currently doesn't allow mutliple dynamic fields (see PDE-948) but when it does, this will account for it
+      let funcNum = 0;
+      fields.forEach((maybeFunc, index) => {
+        if (maybeFunc.source) {
+          const args = maybeFunc.args || ['z', 'bundle'];
+          // always increment the number, but only return a value if it's > 0
+          const funcName = `get${_.upperFirst(key)}${
+            funcNum ? funcNum++ : ++funcNum && ''
```

Comment:
Cool. So it would go as `getInputFields`, `getInputField1`, `getInputFields2`...

Reply:
yep! read then increment. 

```
> let i = 0
undefined
> console.log(i++)
0
undefined
> i
1
> i
1
> console.log(i++)
1
undefined
> i
2
> console.log(++i)
3
undefined
> i
3
> console.log(++i)
4
undefined
> i
4
```